bin/test/main.ml

```diff
@@ -284,7 +284,12 @@ let run_exn (`Setup ()) (`Non_deterministic non_deterministic)
   let syntax =
     match syntax with Some syntax -> Some syntax | None -> Syntax.infer ~file
   in
-  let c = Mdx_top.init ~verbose:(not silent_eval) ~silent ~verbose_findlib () in
+  let c =
+    Mdx_top.init ~verbose:(not silent_eval) ~silent ~verbose_findlib ~dirs:[]
+      ~packages:
+        [ "unix"; "findlib.top"; "findlib.internal"; "compiler-libs.toplevel" ]
```

Comment:
It might be worth extracting this list of packages and the list of directives into some variable somewhere so that it can be reused easily between the mdx binary and the future `mdx.dune_lib` or whatever we'll name it.

Reply:
Good idea, done.